test/cql-pytest/run.py

```diff
@@ -344,7 +344,13 @@ def get_cql_cluster(ip, ssl_context=None):
         # slow debug build running on a very busy machine, they may not be.
         # so let's increase them to 60 seconds. See issue #13239.
         connect_timeout = 60,
-        control_connection_timeout = 60)
+        control_connection_timeout = 60,
+        # The default reconnection policy has a large maximum interval
+        # between retries (600 seconds). In tests that restart/replace nodes,
+        # where a node can be unavailable for an extended period of time,
+        # this can cause the reconnection retry interval to get very large,
+        # longer than a test timeout.
+        reconnection_policy = cassandra.policies.ExponentialReconnectionPolicy(1.0, 4.0))
```

Comment:
What you are changing here is only the connection that run.py uses to check that Scylla came up. The code that is used by actual cql-pytest tests is cql_session() in test/cql-pytest/util.py. 

I don't think this change is important to cql-pytest anyway (which doesn't need to reconnect, Scylla is always assumed to be alive) - so you can drop it from here and not add it to util.py

Reply:
My reasons for copying those parameters to all places were:
- Almost all `Cluster` constructors in `test/` use the same options, so I wanted it to remain consistent
- Setting this particular reconnection policy is non-pessimizing and it's a better default for tests in general
- I wanted to avoid a similar situation as was with `connect_timeout` and `control_connection_timeout`, where those options were originally not added to all places and 4 separate commits at separate times eventually propagated it to all places: 889acf710cac2cc464c74926df84946fe8086c98, d408b711e3f666448412ca7155c3c24360435b77, 4fdcee8415b1b797b9c69ac159a98390199ca8a5, 055340ae39ba7701e798ca370d01d343c28ac231. However, I now see that `connect_timeout` and `control_connection_timeout` are really needed in all tests (for debug build of Scylla), while `reconnection_policy` isn't needed everywhere.